cic/cic.mk

```diff
@@ -56,6 +56,10 @@ PRODUCT_PACKAGES += \
     Browser2 \
     audio.primary.$(TARGET_PRODUCT)
 
+PRODUCT_PACKAGES += health
+
```

Comment:
Do we have health bin?  AFAIK, healthd bin was already included in product package by default

Reply:
this is for addition of health HAL service, which was not enabled by default for CIC project. 
So, added the health service package android.hardware.health@2.0-service.intel here.

Reply:
Good to know:)